RELEASE.md

```diff
@@ -8,6 +8,10 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+## Bug fixes and other changes
+
+- Bug fix pretty name for function names. (#686)
```

Comment:
I think this might not be a very descriptive release note. Could we elaborate a little more?